apps/chat-e2e/src/ui/selectors/dialogSelectors.ts

```diff
@@ -17,6 +17,7 @@ export const PromptModal = {
   promptDescription: '[data-qa="prompt-descr"]',
   promptValue: '[data-qa="prompt-value"]',
   savePrompt: '[data-qa="save-prompt"]',
+  closePrompt: 'svg.tabler-icon.tabler-icon-x',
```

Comment:
'close' icon selector has already been defined in `IconSelectors.cancelIcon`

Reply:
fixed